Workbooks/SapMonitor/SapHanaInfrastructure/SapHanaInfrastructure.workbook

```diff
@@ -0,0 +1,376 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "SapHana_LoadHistory_CL\r\n|summarize arg_max(UTC_TIMESTAMP_t, *) by HOST_s\r\n|project Host=HOST_s , [\"CPU Utilization (%)\"]=CPU_d ,  [\"Memory Utilization (%)\"]=round((MEMORY_USED_d/MEMORY_SIZE_d)*100,2),[\"Disk Utilization (GB)\"]=DISK_USED_d  , [\"Network In\"]=NETWORK_IN_d , [\"Network Out\"]=NETWORK_OUT_d ",
+        "size": 4,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{LogAnalyticsWorkspaceArmId}"
```

Comment:
you have this as a parameter here, but there are no parameters steps.

if this workbook is opened by your blade, and your blade passes in the workspace as `ResourceIds: [ "that workspace id" ]` in the blade parameters, all the query steps will "inherit" that workspace as the default resource.

in that case, you could just leave crossComponentResources empty here in all the steps.

otherwise, one way would be to add a parameters step at the top, and declare a param of type "resource picker", with that name, and have it get its value from the workbook's resources.  then this parameter would be valid here with that name.

Reply:
Great, I'll just leave the `crossComponentResources` section empty then.